airflow/providers/google/cloud/hooks/cloud_sql.py

```diff
@@ -324,21 +315,14 @@ def export_instance(self, instance: str, body: Dict, project_id: Optional[str] =
         :type project_id: str
         :return: None
         """
-        if not project_id:
-            raise ValueError("The project_id should be set")
```

Comment:
This should not be removed I think

Reply:
It can be removed. We have the fallback_to_default_poject_id which will throw an error if project_id cannot be retrieved.